week1/homework/src/server.js

```diff
@@ -8,8 +8,36 @@ const http = require('http');
 function createServer(port) {
   let state = 10;
 
+  function respond(response) {
+    response.writeHead(200, {'Content-Type':'application/json'});
+    response.write(JSON.stringify({state}));
```

Comment:
as a tip: you are using a variable that exists somewhere else in the code.
this works fine, but you want your functions to be as independent as possible to reduce bugs and make them easier to understand.
I would pass `state` to the function 👍